docs/sources/next/results-output/web-dashboard/_index.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Web Dashboard
+excerpt: With the web-dashboard, you can track test results in real-time, and generate HTML test reports directly from your web browser.
+weight: 200
+---
+
+# Web dashboard
+
+k6 provides a built-in web dashboard running on the user's machine and enabling them to monitor their test results in real-time.
+
+## Real-time test results
+
+![Web dashboard screenshot](/media/docs/k6-oss/web-dashboard-overview.png)
+
+The dashboard provides a real-time overview of the performance observed by k6 while a test is running, and helps to identify potential reliability issues as they occur.
+
+## HTML test reports
+
+The dashboard enables to generate detailed, downloadable HTML reports directly from the dashboard. These reports are self-contained, making them ideal for sharing with your team.
+
+![HTML test report screenshot](/media/docs/k6-oss/web-dashboard-report.png)
+
+The HTML test report can be accessed from the dashboard's menu, by clicking on the "Report" button.
+
+![HTML test report generation button](/media/docs/k6-oss/web-dashboard-report-button.png)
+
```

Comment:
Perhaps a sentence could be written about how the report can be automatically saved using the `K6_WEB_DASHBOARD_EXPORT` environment variable, without starting a browser.